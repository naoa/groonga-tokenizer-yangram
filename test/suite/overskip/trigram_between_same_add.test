register tokenizers/yangram

tokenize TokenYaTrigramOverskip "きつつつき" NormalizerAuto --mode ADD
