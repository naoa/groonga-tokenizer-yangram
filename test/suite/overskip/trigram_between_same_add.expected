register tokenizers/yangram
[[0,0.0,0.0],true]
tokenize TokenYaTrigramOverskip "きつつつき" NormalizerAuto --mode ADD
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "きつつ",
      "position": 0
    },
    {
      "value": "つつつ",
      "position": 1
    },
    {
      "value": "つつき",
      "position": 2
    },
    {
      "value": "つき",
      "position": 3
    },
    {
      "value": "き",
      "position": 4
    }
  ]
]
